### PR TITLE
Update changelog for 6.2.0

### PR DESCRIPTION
**Pull request by meghanrjones**
**Description of proposed changes**

Add changes since 6.1.1 to the changelog.

**Preview link**: https://gmt-qwb8g85q9-gmt.vercel.app/changes.html

Relates to #5070 

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Comment by meghanrjones** [comment]
This is slow...I will work on adding the release drafter workflow after finishing this PR.

**Comment by PaulWessel** [comment]
I did not see -w.

**Comment by meghanrjones** [comment]
> I did not see -w.

https://gmt-jgdigywjs-gmt.vercel.app/changes.html#new-common-options-in-gmt-6-2

But, still not done with adding items.

**Comment by PaulWessel** [comment]
Yikes, the premature blindness is worse than I thought.

File: doc/rst/source/changes.rst [hunk 1]
```diff
@@ -4,6 +4,72 @@
...
+#. General improvements to the automatic determination of frame attributes.
+#. New **+d**\ *divisor* modifier to the :ref:`-i option <gmt:The **-i** option>` to simplify scaling of input values.
+#. Allow parsing of **-Jz**\ *1:zzzzzz* for vertical scale.
+#. New GMT configuration parameters :term:`MAP_FRAME_PERCENT`, :term:`COLOR_SET`, and :term:`COLOR_CPT`.
+#. Add support for reading variable in NetCDF-4 groups.
+#. Allow specifying the reciprocal increment for generating 1d arrays.
+#. Allow Latex expressions in single-line titles and Cartesian axes labels, add support for multi-line plot titles, and
+   add support for subtitles.
+#. Many general documentation improvements.
+#. API improvements to support the GMT/MEX, PyGMT, and GMT.jl environments.
```
Comment (joa-quim) [inline doc/rst/source/changes.rst:26]: This is one is too vague and doesn't ring any bell to me. Perhaps drop it

File: doc/rst/source/changes.rst [hunk 2]
```diff
@@ -4,6 +4,72 @@
...
+#. :doc:`histogram`: New **-E** option for custom bar widths and optional shift; New **+b** modifier to **-C** to set
+   color based on the bin value.
+#. :doc:`legend`: New **-M** option to handle both hidden and given information.
+#. :doc:`makecpt`, :doc:`grd2cpt`: Simplify the addition of category labels to CPT files with **-F**.
+#. :doc:`movie`: New modifiers to **-L** and **-P** to enable drop-shadow and rounded rectangular boxes.
+#. :doc:`plot`, :doc:`plot3d`: New **-H** option to scale the symbol size as well as the symbol pen outline attributes;
+   Support sequential auto-colors for polygon fills or line pens.
+#. :doc:`subplot`: New **-D** option to accept previous default plot settings.
+#. :doc:`ternary`: Add support for drawing lines and polygons.
+#. :doc:`text`: New **-S** option to cast shade beneath a text box.
```
Comment (joa-quim) [inline doc/rst/source/changes.rst:60]: We have also the hexagonal plots (sorry, don't remember exactly the details) which are an import improvement.

**Comment by PaulWessel** [comment]
gmt math also got the VPDF operator.
I think for the API you can just say "various improvements to the API in support of developments taking place in the external wrappers (Python, Julia, Matlab).  I cannot really be specific here since mostly low-level stuff.

File: doc/rst/source/changes.rst (hunk 2 above)
Comment (meghanrjones) [reply doc/rst/source/changes.rst:60]: Is this referring to the new gmtbinstats module? PR https://github.com/GenericMappingTools/gmt/pull/3950

**Comment by meghanrjones** [comment]
> gmt math also got the VPDF operator.
> I think for the API you can just say "various improvements to the API in support of developments taking place in the external wrappers (Python, Julia, Matlab). I cannot really be specific here since mostly low-level stuff.

I updated the API bullet point. I think VPDF is included in the list.

File: doc/rst/source/changes.rst (hunk 1 above)
Comment (meghanrjones) [reply doc/rst/source/changes.rst]: I updated with Paul's suggested phrasing.

File: doc/rst/source/changes.rst (hunk 2 above)
Comment (joa-quim) [reply doc/rst/source/changes.rst:60]: Ah, yes. What I had in mind was to emphasize ``..\test\gmtbinstats\hexbinning.ps``

**Comment by PaulWessel** [comment]
> I think VPDF is included in the list.

I only see it under **grdmath**.  There is no separate entry for **gmtmath** I think?

File: doc/rst/source/changes.rst (hunk 2 above)
Comment (meghanrjones) [reply doc/rst/source/changes.rst:61]: OK, I added that gmtbinstats supports both hexagonal and rectangular tiling.

**Comment by meghanrjones** [comment]
> > I think VPDF is included in the list.
> 
> I only see it under **grdmath**. There is no separate entry for **gmtmath** I think?

There wasn't; It is added now.

**Comment by meghanrjones** [comment]
Anything left to do here @PaulWessel?

**Comment by PaulWessel** [comment]
No, I think that is it.  If I have news a bout mex in rc2+ then we add then.

**Comment by meghanrjones** [comment]
> 
> 
> No, I think that is it. If I have news a bout mex in rc2+ then we add then.

Great, can you submit an approval?

**Review by seisman — APPROVED** [review]

**Pull request merged**